packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -346,4 +505,59 @@ contract DeployImplementations is Script {
         dsi_ = DeployImplementationsInput(DeployUtils.toIOAddress(msg.sender, "optimism.DeployImplementationsInput"));
         dso_ = DeployImplementationsOutput(DeployUtils.toIOAddress(msg.sender, "optimism.DeployImplementationsOutput"));
     }
+
+    function deployBytecode(bytes memory _bytecode, bytes32 _salt) public returns (address newContract_) {
+        assembly ("memory-safe") {
+            newContract_ := create2(0, add(_bytecode, 0x20), mload(_bytecode), _salt)
+        }
+        require(newContract_ != address(0), "DeployImplementations: create2 failed");
+    }
+}
+
+contract DeployImplementationsInterop is DeployImplementations {
+    // Overridden because we use a different portal implementation.
+    function deployOptimismPortalImpl(
+        DeployImplementationsInput _dsi,
+        DeployImplementationsOutput _dso
+    )
+        public
+        override
+    {
+        uint256 proofMaturityDelaySeconds = _dsi.proofMaturityDelaySeconds();
+        uint256 disputeGameFinalityDelaySeconds = _dsi.disputeGameFinalityDelaySeconds();
+
+        vm.broadcast(msg.sender);
+        OptimismPortalInterop optimismPortalImpl = new OptimismPortalInterop({
+            _proofMaturityDelaySeconds: proofMaturityDelaySeconds,
+            _disputeGameFinalityDelaySeconds: disputeGameFinalityDelaySeconds
+        });
+
+        vm.label(address(optimismPortalImpl), "OptimismPortalImpl");
+        _dso.set(_dso.optimismPortalImpl.selector, address(optimismPortalImpl));
+    }
+
+    // Overridden because we use a different system config implementation.
+    function deploySystemConfigImpl(DeployImplementationsInput, DeployImplementationsOutput _dso) public override {
+        vm.broadcast(msg.sender);
+        SystemConfig systemConfigImpl = new SystemConfig();
```

Comment:
This is a bug. We also need to add a `deployOPStackManager` override that deploys `OPStackManagerInterop`. I can do this in this PR or a future PR depending on review timelines
```suggestion
        SystemConfig systemConfigImpl = new SystemConfigInterop();
```

Reply:
The `opsmSystemConfigSetter` also needs the new `SystemConfigInterop.selector`

Reply:
Tracking here: https://github.com/ethereum-optimism/optimism/issues/11783